### PR TITLE
Dummy check

### PR DESCRIPTION
**Pull request by elainearbaugh**
Dummy check with get_series() implementation that alternates returning series that will be successful or fail. Also includes a data migration to create the check. All the code is in a new app so we can easily remove it at any point. There are no new views for the check (it can't be edited/deleted) since it's just for testing purposes. 

Check running on stage: 
screen shot 2017-04-10 at 10 07 11 am

### CODE REVIEW

**Comment by elainearbaugh** [comment]
trigger build

File: cabot/dummyapp/models.py [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+        }
+
+        if self.counter % 2 == 0:
+            # Even counts: passing (< 500)
+            series['data'] = [{'series': 'dummy', 'datapoints': [[int(time.time()), 0]]}]
+        else:
+            # Odd counts: failing (> 500)
+            series['data'] = [{'series': 'dummy', 'datapoints': [[int(time.time()), 1000]]}]
+
+        self.counter += 1
```
Comment (mahendra) [inline cabot/dummyapp/models.py]: does this save the model?

File: cabot/metricsapp/models/base.py [hunk 2]
```diff
@@ -0,0 +1,78 @@
+from django.db import models
+from cabot.cabotapp.models import CHECK_TYPES, Service, StatusCheck
+from cabot.metricsapp.api import run_metrics_check
+
+
+class MetricsSourceBase(models.Model):
```
Comment (mahendra) [inline cabot/metricsapp/models/base.py]: isn't this repeated from the other PR's?

Comment (elainearbaugh) [inline cabot/metricsapp/models/base.py]: I haven't merged them yet, changes from only this are here: https://github.com/Affirm/cabot/pull/35/commits/5c0fc52c8ba7a8bf400c122c01ee889d51396b72

I can merge the base models today though

File: cabot/dummyapp/models.py (hunk 1 above)
Comment (elainearbaugh) [inline cabot/dummyapp/models.py]: Added a save(), thanks for catching

**Comment by elainearbaugh** [comment]
trigger build

**Comment by elainearbaugh** [comment]
trigger build

**Comment by elainearbaugh** [comment]
trigger build...

**Pull request merged**